GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -1,157 +1,33 @@
 ﻿using GVFS.Common;
 using GVFS.Common.FileSystem;
-using GVFS.Common.Git;
 using GVFS.Common.Tracing;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.IO.Pipes;
-
+using GVFS.Platform.POSIX;
+
 namespace GVFS.Platform.Mac
 {
-    public partial class MacPlatform : GVFSPlatform
+    public partial class MacPlatform : POSIXPlatform
     {
         public MacPlatform()
             : base(
-                executableExtension: string.Empty,
-                installerExtension: ".dmg",
-                underConstruction: new UnderConstructionFlags(
-                    supportsGVFSService: false,
-                    supportsGVFSUpgrade: false,
-                    supportsGVFSConfig: false))
+                installerExtension: "." + InstallerExtension)

```

Comment:
It looks like this is now the only spot in the code where `InstallerExtension` is being used (after removing `GetInstallerExtension` from `GVFSHooksPlatform`).  

Could you remove `InstallerExtension`  or update the extension string to include the `.`?